src/ape/api/transactions.py

```diff
@@ -364,7 +364,7 @@ def await_confirmations(self) -> "ReceiptAPI":
         if self._explorer:
             explorer_url = self._explorer.get_transaction_url(self.txn_hash)
             if explorer_url:
-                log_message = f"{log_message}\n{self._explorer.name} URL: {explorer_url}"
+                log_message = f"Submitted {explorer_url}"
 
         logger.info(log_message)
```

Comment:
(can't get the line above this)

but this might look a little slimmer:
```py
        if explorer_url := self._explorer and self._explorer.get_transaction_url(self.txn_hash):
            logger.info(f"Submitted {explorer_url}")
        else:
            logger.info(f"Submitted {self.txn_hash}")
```